src/App.css

```diff
@@ -16,7 +16,7 @@ body {
 }
```

Comment:
- [ ] OPTIONAL: I recommend aligning your project to the center.
- [ ] OPTIONAL: I recommend using the same fonts as in the template.
- [ ] OPTIONAL: I recommend fixing the "categories" part of the project, there are some overflows.

![image](https://user-images.githubusercontent.com/57421823/155629757-6530f422-d103-49b8-9ee9-1d52e0b3ad2b.png)

![image](https://user-images.githubusercontent.com/57421823/155629903-5b0baba0-c8d9-4df6-b195-612468a7a667.png)

_Tested on Firefox developer edition._